includes/Story_Post_Type.php

```diff
@@ -774,7 +721,6 @@ public static function get_schemaorg_metadata() {
 			'publisher' => [
 				'@type' => 'Organization',
 				'name'  => $publisher['name'],
-				'logo'  => $publisher['logo'],
```

Comment:
Removed publisher logo from config since now we can actually update this field and it has to be up to date when creating output for a Story. Using the assigned URL directly now.

Reply:
This affects schema.org metadata output when rendering the story.

We could use [`react-schemaorg`](https://www.npmjs.com/package/react-schemaorg) to print the schema.org metadata (at least parts of it) on the client-side.

Then, we could update `Story_Renderer` / `Story_Post_Type::print_schemaorg_metadata` to merge the generated data with the one provided by PHP, and to make it filterable by plugins.

Thoughts?

Reply:
Hmm, good point.

This might not be a problem on the server-side actually, just restoring it should be enough. It was a problem while creating the markup for the Story since the story markup is using the previously saved value that came from the server, however, when generating schema.org metadata on the server-side, the value should be up to date already, it's displayed dynamically anyway. Will just restore this.